lib/ultrajsondec.c

```diff
@@ -169,7 +171,7 @@ static FASTCALL_ATTR JSOBJ FASTCALL_MSVC decode_numeric (struct DecoderState *ds
           }
           else if (intNeg == -1)
           {
-            return SetError(ds, -1, overflowLimit == LLONG_MAX ? "Value is too big!" : "Value is too small");
+            return SetError(ds, -1, overflowLimit == LLONG_MAX ? "Value is too big" : "Value is too small");
           }
```

Comment:
Is the `overflowLimit == LLONG_MAX ? ...` switch not redundant because of the `if (intNeg == 1)` and `else if (intNeg == -1)` conditions above it?

Reply:
(P.S. If we're ditching `!`s in error messages then there's that one still lurking on line 170.